src/_color-themes.scss

```diff
@@ -0,0 +1,103 @@
+// Themes
+
+$bg: "bg";
+$fg: "fg";
+$lightest-blue: "lightest-blue";
+$lighter-blue: "lighter-blue";
+$light-blue: "light-blue";
+$blue-custom: "blue-custom";
```

Comment:
What is the `-custom` suffix for here?

Reply:
I think what happened was that the mapping got confused by names like `$teal`,`$black` etc. because they are standard colors and the theming wouldn't apply. When applying the renaming mentioned before this would become unnecessary though.